Components/BlockImageTextParallax/script.js

```diff
@@ -0,0 +1,51 @@
+import Rellax from 'rellax'
+import $ from 'jquery'
+
+const $window = $(window)
+
+let rellax, initialized
+
+class BlockImageTextParallax extends window.HTMLDivElement {
+  constructor (self) {
+    self = super(self)
+    self.$ = $(self)
+    return self
```

Comment:
Please replace constructor with new best practices

Reply:
_ignored this one as agreed_